Gemfile

```diff
@@ -47,6 +47,7 @@ group :development, :test do
   gem 'rspec-rails'
   gem 'dotenv-rails'
   gem 'pry-byebug'
+  gem 'rubocop', require: false
```

Comment:
we should use `0.74`, as it's the most recent version supported by code climate.